### PR TITLE
[13.0][FIX] stock_account: post-migration: fill layer usages for dropshipments

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
I do this in a separate method because in the other one is a nightmare and I don't see any performance drawback, on the contrary, doing this in the same method could be tricky and lousy :laughing: 

cc @MiquelRForgeFlow

### CODE REVIEW

File: addons/stock_account/migrations/13.0.1.1/post-migration.py [hunk 1]
```diff
@@ -373,11 +373,44 @@ def generate_stock_valuation_layer(env):
         query_insert(env.cr, "stock_valuation_layer_usage", all_svl_usage_list)
 
 
+def fill_svl_usage_dropshipments(env):
+    """After generate_stock_valuation_layer dropship layers are not linked"""
+    svl_usage_vals_list = []
+    openupgrade.logged_query(
+        env.cr,
+        """
+            WITH BQ AS (
```
Comment (MiquelRForgeFlow) [inline addons/stock_account/migrations/13.0.1.1/post-migration.py]: what means BQ?

Comment (AaronHForgeFlow) [reply addons/stock_account/migrations/13.0.1.1/post-migration.py]: Base query

**Comment by AaronHForgeFlow** [comment]
don't merge yet, I can do it better

**Comment by AaronHForgeFlow** [comment]
Now it is better, because it uses source and destination locations, same as the other query, and also checks there is no usage, in order to prevent duplicated usages. Now it is ready IMHO.

**Comment by AaronHForgeFlow** [comment]
please_merge

**Pull request merged**